app/src/main/java/gis2018/udacity/tametu/utils/Utils.java

```diff
@@ -51,7 +51,8 @@ public static int updateWorkSessionCount(SharedPreferences preferences, Context
      */
     public static int getTypeOfBreak(SharedPreferences preferences, Context context) {
         int currentWorkSessionCount = preferences.getInt(context.getString(R.string.work_session_count_key), 0);
-        if (currentWorkSessionCount % 4 == 0)
+        int session= preferences.getInt(context.getString(R.string.start_long_break_after), 2);
```

Comment:
This merely fetches the positional values, e.g. 0, 1, 2, 3 etc. For this reason when _2 sessions_ is selected, the app crashes. Also, it doesn't work as expected when selecting any other values.

Values must be assigned to such selections. e.g. https://github.com/NJACKWinterOfCode/android-productivity-timer-tametu-app/blob/40f508ab73eb758924e15d1a5686e4a91e4b131b/app/src/main/java/gis2018/udacity/tametu/utils/Utils.java#L91

Also consider renaming `session` to something like `longBreakIntervalSessions`.